### PR TITLE
Edit email confirmation text

### LINKED ISSUE #30: website - order confirmation email
email sent with order confirmation after order is submitted.
relies on email server being set up first

### PR DESCRIPTION
**Pull request by Lfickling**
added in confirmation email text. some small web-page edits, esp to thank you page. some were things i started on but had some issues with so i commented out (mostly time delay redirects).
closes #30

### CODE REVIEW

File: src/Website/Resources/orderconfirmationemail.php [hunk 1]
```diff
@@ -23,9 +23,17 @@
 //$mail->addAttachment('/tmp/image.jpg', 'new.jpg');    // Optional name
 $mail->isHTML(true);                                  // Set email format to HTML
 
-$mail->Subject = 'Order Confirmation';
-$mail->Body    = $_SESSION["email"];
-$mail->AltBody = $_SESSION["quantity"];
+$mail->Subject = 'Shark Order Confirmation';
```
Comment (jgonzo3224) [inline src/Website/Resources/orderconfirmationemail.php:26]: looks good and thorough text added to the confirmation email.

File: src/Website/Resources/register.php [hunk 2]
```diff
@@ -62,17 +62,15 @@
...
                 } else{
                     echo "Something went wrong. Please try again later.";
                 }
-    
-                // Close statement
-                mysqli_stmt_close($stmt);
+            // Close statement
+            mysqli_stmt_close($stmt);
             }
-
```
Comment (jgonzo3224) [inline src/Website/Resources/register.php:74]: code cleanup looks good.

File: src/Website/Resources/thankyou.php [hunk 3]
```diff
@@ -43,28 +43,21 @@
...
     </div>
     <div class="content-wrapper">
        <div class="order-form-wrapper">
             <h1>Thank you for your order!</h1>
-            <h1>Your order of </h1><?php echo htmlspecialchars($_SESSION["quantity"]); ?><h1> </h1><?php echo htmlspecialchars($_SESSION["itemid"]); ?><h1>is being processed</h1>
+            <h1>Your order of <?php echo htmlspecialchars($_SESSION["quantity"]); ?> <?php echo htmlspecialchars($_SESSION["itemid"]); ?> is being processed</h1>
+       </div>
+       <h1>\nTo place another order click <a href="order.php">here</a> </h1>
+       <div class="portfolio-item-wrapper">
+           <div class="portfolio-img-background" style="background-image:url(images/sharkdab.jpg)"></div>
```
Comment (jgonzo3224) [inline src/Website/Resources/thankyou.php:59]: Added some style with that Shark Dab image.

**Review by jgonzo3224 — APPROVED** [review]
Good code cleanup and organization along with adding an image and changing some of the other images.

File: src/Website/Resources/thankyou.php (hunk 3 above)
Comment (Lfickling) [reply src/Website/Resources/thankyou.php:59]: hell yeah!

**Comment by DARhine** [comment]
Looks great. Nice job.

**Review by DARhine — APPROVED** [review]
Looks good. Nice clean up effort.

**Review by jgonzo3224 — APPROVED** [review]

**Review by DARhine — APPROVED** [review]
Awesome! Looks good, nice code clean up.

**Comment by DARhine** [comment]
Merging pull. Edit of email and code clean up.

**Pull request merged**